packages/vite/src/node/publicUtils.ts

```diff
@@ -6,6 +6,7 @@
 export { VERSION as version } from './constants'
 export { version as esbuildVersion } from 'esbuild'
 export { VERSION as rollupVersion } from 'rollup'
+export { DEFAULT_DEV_PORT, DEFAULT_PREVIEW_PORT } from './constants'
```

Comment:
I don't think we should re-export this. Vite has many defaults and it wouldn't be scalable to re-export all of them (even though we're starting with one).

I'd say we should revert this for now so we can get the refactors in.